roboteam_ai/src/skills/Attack.cpp

```diff
@@ -23,17 +23,25 @@ bt::Node::Status Attack::onUpdate() {
     Vector2 ball = World::getBall().pos;
     Vector2 behindBall = Coach::getPositionBehindBall(0.5);
 
-    if (!Control::pointInTriangle(robot->pos, ball, ball + (behindBall-ball).rotate(M_PI*0.3).scale(1.5),
-            ball + (behindBall-ball).rotate(M_PI*-0.3).scale(1.5))) {
+    if (!Control::pointInTriangle(robot->pos, ball, ball + (behindBall-ball).rotate(M_PI*0.17).scale(1.74),
+            ball + (behindBall-ball).rotate(M_PI*-0.17).scale(1.74))) {
         targetPos = behindBall;
+        goToPos.goToPos(robot, targetPos, GoToType::luTh);
     } else if (!Coach::doesRobotHaveBall(robot->id, true)) {
         targetPos = ball;
+        goToPos.goToPos(robot, targetPos, GoToType::basic);
     } else {
         targetPos = ball;
         unsigned char forced_kick = 1;
         kicker.kick(robot, forced_kick);
+        goToPos.goToPos(robot, targetPos, GoToType::basic);
```

Comment:
Why is this added?

Reply:
basic gebruik ik voor low level control, luth voor high level / grote afstand

Reply:
I mean the lines below it. looks like you added extra functionality.

Reply:
Those additions then don't belong in this PR. This PR is about removing a PID controller. (might you be asking why: later on this PR will be converted to one big commit on development, thus making it difficult to trace an issue if it would be there)